docs/sample_config.yaml

```diff
@@ -1422,6 +1424,14 @@ metrics_flags:
 #  - "m.room.encryption"
 #  - "m.room.name"
 
+# Uncomment to enable experimental room knocking support as defined by
+# MSC2403.
+#
+# Note that the APIs used by this feature are unstable and will break in the
+# future.
+#
+#msc2403_enabled: true
```

Comment:
If we do this a lot it might make sense to come up with a more standard way. (I think dendrite has an MSC section. 🤷 ) This seems OK for now though.

Reply:
That's the plan, but one that's still in discussion. Asking in #synapse-dev and looks like we're going to leave this as is.

Reply:
We have a more standard way at this point ([added in #9183](https://github.com/matrix-org/synapse/pull/9183/files#diff-3af529eedb0e00279bafb7369370c9654b37792af8eafa0925400e9281d57f0a)), so I'll update this PR to include that.